js/WebData.js

```diff
@@ -0,0 +1,61 @@
+export default class WebData {
+    constructor() {
+        // web api for battery level
+        this.batteryPercentage = 0;
+        this.getBatteryStatus();
+        this.updateTime();
+    }
+
+    updateBatteryStatus(battery) {
+        const level = document.getElementById('level');
+        const batteryLevelDiv = document.getElementById('battery-level');
+
+        const percentage = Math.round(battery.level * 100);
+        level.textContent = percentage + '%';
+
+        // Update the battery level div width and color
+        batteryLevelDiv.style.width = percentage + '%';
+
+        if (percentage > 50) {
+            batteryLevelDiv.style.backgroundColor = 'green';
```

Comment:
Je hebt in de CSS de kleur aangepast, maar hier nog niet.

Reply:
lightgreen